### PR TITLE
RNs-4.9.41 Added with bug fix update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.9.41**
Added with bug fix update

**Version(s):**
PR applies only to enterprise-4.9

**Issue:**
https://issues.redhat.com/browse/OCPPLAN-9498

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.9.41/release_notes/ocp-4-9-release-notes.html#ocp-4-9-41

**Additional information:**

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2941,3 +2941,26 @@ link:https://access.redhat.com/solutions/6964719[{product-title} 4.9.40 containe
...
+{product-title} release 4.9.41, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5434[RHBA-2022:5434] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5433[RHBA-2022:5433] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6965553[{product-title} 4.9.41 container image list]
+
+[id="ocp-4-9-41-bug-fixes"]
+==== Bug fixes
+
+* Previously, the HAProxy redirected the wrong HTTPS route rather than the more detailed one, preventing users from having their entire application validated without being redirected to HTTPS. This fix sets a flag on the redirect map, ensuring the HAProxy redirects the appropriate HTTPS route. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010227#[*BZ#2010227*])
```
Comment (michaelryanpeter) [inline release_notes/ocp-4-9-release-notes.adoc:2959]:
```suggestion
* Previously, HAProxy redirected to the wrong HTTPS route. As a result, applications were not redirected to HTTPS and were not validated. This fix sets a flag on the redirect map to ensure that HAProxy redirects the correct HTTPS route. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010227#[*BZ#2010227*])
```
This is a little difficult to follow. Does this suggestion change the meaning too much?

**Review by michaelryanpeter — COMMENTED** [review]
Please let me know if you have questions!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: @michaelryanpeter Not at all! This is great! Thank you so much for your help! :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2941,3 +2941,26 @@ link:https://access.redhat.com/solutions/6964719[{product-title} 4.9.40 containe
...
+
+* Previously, when starting the PipelineRun creation with the VolumeClaimTemplates, the pipelines created used the hard-coded value (gp2) rather than the assigned `StorageClass` name. This fix uses the specified `StorageClass` name rather than the hard-coded value, resulting in the appropriate `StoragClass` name assigned when starting a pipeline with the VolumeClaimTemplate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097618#[*BZ#2097618*])
```
[the remark this answers is not in the document]
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Hi @michaelryanpeter ,

Thank you for your help! I should've reviewed the guidance beforehand, but I have made the correct updates! I did utilize the `PipelineRun` and `volumeClaimTemplate`; although I can change it if need be. :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2941,3 +2941,26 @@ link:https://access.redhat.com/solutions/6964719[{product-title} 4.9.40 containe
...
+{product-title} release 4.9.41, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5434[RHBA-2022:5434] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5433[RHBA-2022:5433] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6965553[{product-title} 4.9.41 container image list]
+
+[id="ocp-4-9-41-bug-fixes"]
+==== Bug fixes
+
+* Previously, when a `PipelineRun` was created with a `volumeClaimTemplate`, the pipelines used the hard-coded value of `gp2` rather than the assigned storage class name. With this fix, the appropriate storage class name is assigned when starting a `PipelineRun` with a `volumeClaimTemplate`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097618#[*BZ#2097618*])
```
[the remark this answers is not in the document]
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Done! Thanks for catching! :)

**Pull request merged**